main.c

```diff
@@ -1322,7 +1322,7 @@ main
     {
       const bool c_imap_passive = cs_subset_bool(NeoMutt->sub, "imap_passive");
       cs_subset_str_native_set(NeoMutt->sub, "imap_passive", false, NULL);
-      const CheckStatsFlags csflags = MUTT_MAILBOX_CHECK_FORCE | MUTT_MAILBOX_CHECK_IMMEDIATE;
```

Comment:
Why do we no longer want to update the postponed number on startup?

Reply:
I don't think that was ever needed. The branch here is to handle `-Z`, which is meant to open the first mailbox with new mail. We're only interested in new mail checks here.

Reply:
Ah, I didn't check which main branch we where in. Yep, makes sense.